test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -582,10 +582,10 @@ private int GetMaxIndexForOnePrediction(MNISTPrediction onePrediction)
 
         public class MNISTData
         {
-            [Column("0")]
+            [LoadColumn("0")]
             public long Label;
 
-            [Column(ordinal: "1-784")]
+            [LoadColumn(range: "1-784")]
```

Comment:
>LoadColumn [](start = 13, length = 10)

I think we need some good word smith here.
It's probably slightly better than just column, but still confusing, especially next to that ColumnName attribute few lines below.

Can it be TextLoader.MarkingColumn? TextLoader.MappingColumn, something related to textloader since it's used only for Textloader (right?)

Reply:
It is called LoadColumn, if you are referring to the new attribute. 

---
In reply to: [241852931](https://github.com/dotnet/machinelearning/pull/1878#discussion_r241852931) [](ancestors = 241852931)